CHANGELOG.rst

```diff
@@ -0,0 +1,49 @@
+OMEGO version history
+=====================
+
+0.3.0 (February 2016)
+---------------------
+
+ * drops support for OMERO 5.1 and earlier
```

Comment:
(Though no objection to saying we will be dropping support for installing pre-5.2, but upgrading from older version will still be possible for now)